app/controllers/packages_controller.rb

```diff
@@ -0,0 +1,63 @@
+class PackagesController < ApplicationController
+  before_action :authenticate_user!
+  def index
+    @list_of_packages = current_user.packages.order(created_at: :desc)
+
+    render({ :template => "packages/index" })
+  end
+
+  def show
+    the_id = params.fetch("path_id")
+
+    matching_packages = Package.where({ :id => the_id })
+
+    @the_package = matching_packages.at(0)
+
+    render({ :template => "packages/show" })
+  end
+
+  def create
+    the_package = Package.new
+    the_package.description = params.fetch("query_description")
+    the_package.estimated_arrival = params.fetch("query_estimated_arrival")
+    the_package.details = params.fetch("query_details")
+    #the_package.tracking = params.fetch("query_tracking")
+    the_package.recieved = params.fetch("query_recieved", false)
```

Comment:
You could probably set this to false without having to query anything if a users package is automatically set to false when they log it